pkg/validators/validators.go

```diff
@@ -23,7 +23,14 @@ import (
 	"github.com/zclconf/go-cty/cty"
 )
 
-const projectError = "project ID %s does not exist or your credentials do not have permission to access it"
+func projectError(p string) error {
+	return config.HintError{
+		Err: fmt.Errorf("project %q does not exist or your credentials do not have permission to access it", p),
+		Hint: "It may be the machine you are working on has not been authenticated.\n" +
```

Comment:
nit: I think it is fine as is. Possible improvement:

**It is possible** the machine you are working on has not been authenticated.